QModManager/Patching/Patcher.cs

```diff
@@ -166,8 +211,8 @@ private static void AddAssemblyResolveEvent()
 
         private static void PatchHarmony()
         {
-            var harmony = HarmonyInstance.Create("qmodmanager");
-            harmony.PatchAll();
+            Logger.Debug("Applying Harmony patches...");
+            HarmonyInstance.Create("qmodmanager").PatchAll();
```

Comment:
We discussed wrapping the `PatchAll()` request within a try-catch for extra safety.